src/SourceGenerators/Uno.UI.SourceGenerators/XamlGenerator/XamlFileGenerator.cs

```diff
@@ -3519,38 +3519,30 @@ private ITypeSymbol GetXBindPropertyPathType(string propertyPath, INamedTypeSymb
 		private bool IsStaticMember(string fullMemberName)
 		{
 			fullMemberName = fullMemberName.TrimStart("global::");
-
 			var lastDotIndex = fullMemberName.LastIndexOf(".");
 
-			if (lastDotIndex != -1)
-			{
-				var className = lastDotIndex != -1 ? fullMemberName.Substring(0, lastDotIndex) : fullMemberName;
-				var memberName = lastDotIndex != -1 ? fullMemberName.Substring(lastDotIndex + 1) : fullMemberName;
+			var isInstance = lastDotIndex == -1;
```

Comment:
I think this does not denote an instance member, but rather an immediate member of the current scope. `isInstance` is probably misleading.

Reply:
I took a shot at renaming this to something that makes sense but I'm struggling to think of a good name

Reply:
Same ;) Maybe let's go for `isTopLevelMember` ?